### PR TITLE
Bug 1600025 - Branding Updates for OpenShift

### PR DESCRIPTION
**Pull request by spadgett**
* Default to OpenShift Origin branding if not specified
* Change flag name from --logo-image-name to --branding
* Update document title based on branding

https://bugzilla.redhat.com/show_bug.cgi?id=1600025

### CODE REVIEW

File: frontend/public/index.html [hunk 1]
```diff
@@ -5,7 +5,18 @@
     <base href="[[ .BasePath ]]">
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
-    <title>Tectonic Console</title>
+    [[ if eq .LogoImageName "origin" ]]
```
Comment (spadgett) [inline frontend/public/index.html]: I'm considering renaming `--logo-image-name` this to something more general like `--edition`

File: frontend/public/components/app.jsx [hunk 2]
```diff
@@ -135,9 +135,27 @@ class App extends React.PureComponent {
...
+      case 'origin':
+        return 'OpenShift Origin';
+      case 'ocp':
+        return 'OpenShift Container Platform';
+      case 'online':
+        return 'OpenShift Online';
+      case 'tectonic':
+        return 'Tectonic';
+      default:
+        return null;
```
Comment (spadgett) [inline frontend/public/components/app.jsx]: @alecmerdler Would you object to making the default branding be OpenShift in the master branch and setting `BRIDGE_LOGO_IMAGE_NAME=tectonic` for Tectonic installs?

File: frontend/public/index.html (hunk 1 above)
Comment (benjaminapetersen) [reply frontend/public/index.html]: IS 'edition' a term that is becoming popular?

File: frontend/public/components/app.jsx (hunk 2 above)
Comment (alecmerdler) [reply frontend/public/components/app.jsx]: I would prefer the default make sense when running against pure upstream Kubernetes, but I think we just fallback to Tectonic logo currently.

File: frontend/public/index.html (hunk 1 above)
Comment (spadgett) [reply frontend/public/index.html]: i don't know if that's the best term, maybe `branding`?

File: frontend/public/components/app.jsx (hunk 2 above)
Comment (spadgett) [reply frontend/public/components/app.jsx]: Fair, but yeah it currently defaults to Tectonic, which isn't neutral either

Comment (alecmerdler) [reply frontend/public/components/app.jsx]: @spadgett Best option would be to not render the top bar if running on pure upstream Kubernetes.

Comment (spadgett) [reply frontend/public/components/app.jsx]: It's more than that. There are favicons, doc links, and branding in other parts of the console.

File: frontend/public/index.html (hunk 1 above)
Comment (benjaminapetersen) [reply frontend/public/index.html]: `branding` is pretty clear, I tend toward that preference.

**Comment by spadgett** [comment]
@alecmerdler PTAL

**Comment by spadgett** [comment]
jenkins push

**Comment by coreos-ui** [comment]
The PR image has been pushed to Quay with tag `pr_217_build_402`

**Comment by spadgett** [comment]
/unassign @TheRealJon 

Thanks @benjaminapetersen

**Pull request merged**